### PR TITLE
Add com.ibm.ws.org.apache.cxf.cxf.api.2.6 bundle

### PR DESCRIPTION
**Pull request by neuwerk**
This bundle is being created to split up the com.ibm.ws.org.apache.cxf.jaxws and child bundles as well as add the open-source changes that have gone into our version of CXF.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/com/ibm/ws/jaxws23/cxf/api/NoOpActivator.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.api;
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/com/ibm/ws/jaxws23/cxf/api/NoOpActivator.java]: jaxws22

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/com/ibm/ws/jaxws23/cxf/api/package-info.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.api;
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/com/ibm/ws/jaxws23/cxf/api/package-info.java]: 22

**Comment by andrea-pichler** [comment]
Hi @neuwerk 
dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties shows
INJECTION_SETTER_RAISED_EXCEPTION=exception **raied** when invoking setter {0}
->this should be **raised** I think
Message review completed by L2

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#    software distributed under the License is distributed on an
+#    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#    KIND, either express or implied. See the License for the
+#    specific language governing permissions and limitations
+#    under the License.
+#
+#
+RESOURCE_RESOLVE_FAILED=failed to resolve resource {0}
+FAILED_TO_INJECT_FIELD=access to field is not allowed
+INJECTION_SETTER_NOT_VISIBLE=setter {0} is not visible
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
setter {0} is not visible
**to**
The {0} setter is not visible.

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+INJECTION_SETTER_RAISED_EXCEPTION=exception raised when invoking setter {0}
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
exception raised when invoking setter {0}
**to**
An exception was raised when invoking the {0} setter.

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+PRIVATE_FIELD_INJECTION_NYI=resource injection for private fields is not yet implemented: {0}
+INJECTION_COMPLETE_NOT_VISIBLE=method annotated by @PostConstruct is not visible: {0}
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
method annotated by @PostConstruct is not visible: {0}
**to**
The method that is annotated by @PostConstruct is not visible: {0}

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+INJECTION_COMPLETE_THREW_EXCEPTION=method annotated by @PostConstruct throws exception when invoked
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
method annotated by @PostConstruct throws exception when invoked
**to**
The method that is annotated by @PostConstruct throws an exception when invoked.

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+PRE_DESTROY_NOT_VISIBLE=Method annotated by @PreDestroy is not visible: {0}
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
Method annotated by @PreDestroy is not visible: {0}
**to**
The method that is annotated by @PreDestroy is not visible: {0}

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+PRE_DESTROY_THREW_EXCEPTION=Method annotated by @PreDestroy throws exception when invoked
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
Method annotated by @PreDestroy throws exception when invoked
**to**
The method that is annotated by @PreDestroy throws an exception when invoked.

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+SETTER_INJECTION_WITH_INCORRECT_TYPE=setter for resource found but with wrong signature: {0}
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
setter for resource found but with wrong signature: {0}
**to**
The setter for the resource was found but with wrong signature: {0}

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+RESOURCE_NAME_NOT_SPECIFIED=Resource annotation on {0} must specify resource name
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
Resource annotation on {0} must specify resource name
**to**
The resource annotation on {0} must specify a resource name.

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+NO_SETTER_OR_FIELD_FOR_RESOURCE=Resource annotation on {0} but no field or setter found.
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
Resource annotation on {0} but no field or setter found.
**to**
No field or setter was found for the resource annotation on {0}.

File: dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+INJECTION_SETTER_METHOD_NOT_FOUND = Can''t find the injection setter method: {0}
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties]: **change**
Can''t find the injection setter method: {0}
**to**
Can't find the injection setter method: {0}

**Comment by Charlotte-Holt** [comment]
Based on another messages PR that was Apache-related, these messages aren't changeable. Adding ID review not needed label.

**Pull request merged**